content/src/debug.js

```diff
@@ -201,38 +364,205 @@ function reportSeeeval(method, debugId, length, coordId, elem, args){
   debugRecordsByDebugId[currentDebugId] = record;
 }
 
+function reportEnter(method, debugId, length, coordId, elem, args){
+  stuckComplexity.calls += 1;
+  var recordD = debugRecordsByDebugId[debugId];
+  var recordL = debugRecordsByLineNo[recordD.line];
```

Comment:
I don't think we need to assign to recordL here - we should be able to treat debugRecordsByLineNo as a read-only collection (except for when new mappings are added).